core/src/main/java/com/novoda/noplayer/PlayerState.java

```diff
@@ -65,6 +65,16 @@
      */
     long playheadPositionInMillis();
```

Comment:
Should we deprecate this old method then? It seems like having a _convenience_ method like this will only lead to subtle errors.

Reply:
If it is ok with you I'd rather deprecate it when we have all of the advert specific ones in place as well. As it is we have a mismatch 😄

Reply:
That's fine, I thought having advert break methods is enough, but it's true that we should probably expose advert-specific methods too (... at some point 😄)